src/app/nextversion/nextversion.go

```diff
@@ -17,11 +19,13 @@ import (
 )
 
 const (
-	outputPrefix = "output-prefix"
-	tagPrefix    = "tag-prefix"
-	currentFlag  = "current"
-	nextFlag     = "next"
-	gitRootFlag  = "git-root"
+	outputPrefix              = "output-prefix"
+	tagPrefix                 = "tag-prefix"
+	currentFlag               = "current"
+	nextFlag                  = "next"
+	gitRootFlag               = "git-root"
+	limitVersionBumpToFlag    = "limit-version-bump-to"
+	limitDependencyBumpToFlag = "limit-dependency-bump-to"
```

Comment:
These seem a bit long arg names. Could we pick a bit shorter ones?
```suggestion
	limitVersionBumpToFlag    = "bump-cap"
	limitDependencyBumpToFlag = "dependency-cap"
```
I think we should be consistent with the naming, in this case "cap". If we prefer to call it limit then I think we should change it everywhere (including the code).

Reply:
Solved in [#2cb3566](https://github.com/newrelic/release-toolkit/pull/128/commits/2cb3566e752425a9d12752ca6a5388b3461d09c4)